### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by martoio** [iliazakharov/prj-rev-bwfs-dasmoto#5]
Make sure you add the ``alt`` attribute to your images to maintain accessibility for all audiences of the page (particularly visually-impaired users). Remember to add it even in these simple projects and it will become a habit. (Hint: as long as you have the ``alt`` attribute with something semi-meaningful, we'll be happy with it).
For more info: https://www.codecademy.com/articles/accessibility

Also, your syntax for the image tag is incorrect:
``<img src="..."  alt="..."/>``
https://github.com/iliazakharov/prj-rev-bwfs-dasmoto/blob/c7449ddc7dbbc238d73123fbbac95d7dc71aaab9/Dasmato's/index.html#L13

**Comment by iliazakharov** [comment]
You are right , i totally missed that point. Thank you for pointing that out.
